venues/ICLR.cc/2020/Conference/python/buddyAreachairWebfield.js

```diff
@@ -0,0 +1,984 @@
+        paperNumber: note.number,
+        reviewerNumber: reviewerNum
+      };
+    }
+  }
+
+  var cell2 = {
+    noteId: note.id,
+    paperNumber: note.number,
+    numSubmittedReviews: Object.keys(completedReviews).length,
+    numReviewers: Object.keys(reviewerIds).length,
+    reviewers: combinedObj,
+    sendReminder: true,
+    expandReviewerList: false,
+    enableReviewerReassignment : ENABLE_REVIEWER_REASSIGNMENT
+  };
+  reviewerSummaryMap[note.number] = cell2;
+
+  // Rating cell
+  var cell3 = {
+    averageRating: 'N/A',
+    minRating: 'N/A',
+    maxRating: 'N/A'
+  };
+  if (ratings.length) {
+    cell3.averageRating = _.round(_.sum(ratings) / ratings.length, 2);
+    cell3.minRating = _.min(ratings);
+    cell3.maxRating = _.max(ratings);
+  }
+
+  // Confidence cell
+  var cell4 = {
+    averageConfidence: 'N/A',
+    minConfidence: 'N/A',
+    maxConfidence: 'N/A'
+  };
+  if (confidences.length) {
+    cell4.averageConfidence = _.round(_.sum(confidences) / confidences.length, 2);
+    cell4.minConfidence = _.min(confidences);
+    cell4.maxConfidence = _.max(confidences);
+  }
+
+  // Status cell
+  var invitationUrlParams = {
+    id: note.forum,
+    noteId: note.id,
+    invitationId: getInvitationId('Meta_Review', note.number)
+  };
+  var cell5 = {
+    invitationUrl: '/forum?' + $.param(invitationUrlParams)
+  };
+  if (metaReview) {
+    cell5.recommendation = metaReview.content.recommendation;
+    cell5.editUrl = '/forum?id=' + note.forum + '&noteId=' + metaReview.id;
+  }
+
+  return [cellCheck, cell0, cell1, cell2, cell3, cell4, cell5];
+};
+
+var findNextAnonGroupNumber = function(paperNumber){
+  paperReviewerNums = Object.keys(reviewerSummaryMap[paperNumber].reviewers).sort();
+  for (var i = 1; i < paperReviewerNums.length + 1; i++) {
+    if (i.toString() !== paperReviewerNums[i-1]) {
+      return i;
+    }
+  }
+  return paperReviewerNums.length + 1;
+}
+
+// Event Handlers
+var registerEventHandlers = function() {
+  $('#group-container').on('click', 'a.note-contents-toggle', function(e) {
+    var hiddenText = 'Show paper details';
+    var visibleText = 'Hide paper details';
+    var updated = $(this).text() === hiddenText ? visibleText : hiddenText;
+    $(this).text(updated);
+  });
+
+  $('#group-container').on('click', 'a.send-reminder-link', function(e) {
+    var $link = $(this);
+    var userId = $link.data('userId');
+    var forumUrl = $link.data('forumUrl');
+
+    var sendReviewerReminderEmails = function(e) {
+      var postData = {
+        groups: [userId],
+        forumUrl: forumUrl,
+        subject: $('#message-reviewers-modal input[name="subject"]').val().trim(),
+        message: $('#message-reviewers-modal textarea[name="message"]').val().trim(),
+      };
+
+      $('#message-reviewers-modal').modal('hide');
+      // promptMessage('Your reminder email has been sent to ' + view.prettyId(userId));
```

Comment:
In the master branch, this line is uncommented. If it's not needed here maybe it should be removed

Reply:
Synchronized with master. Thanks for noticing this.